### DISCUSSION TITLE
Task 3 lab 4.02

### DISCUSSION THREAD

**Discussion by davidlon13** [rramosp/ai4eng.v1#168] [04 - GESTIÓN DE DATOS]
Hola saludo cordial.

Tengo una inquietud con la tarea 3 del Lab 4.02, sucede que creo estar cargando la respuesta correcta, y el sistema me sigue calificando cero con el siguiente mensaje.

task_03 submitted. your grade is 0
----- grader message -------
checking for date='2016-02-02'
incorrect X or y dimensions

En esta tarea nos piden tomar la serie temporal de la tarea 2 donde se fijaron los valores pedidos y según sus valores y sus dimensiones crear una matriz X y un vector Y. En mi caso la serie temporal contiene 24 puntos por lo tanto las dimensiones X serán (20,3) y las dimensiones y serán (20). 

Mi respuesta para X fue:
[[8.0, 8.0, 8.0],
 [8.0, 8.0, 8.0],
 [8.0, 8.0, 8.0],
 [8.0, 8.0, 8.0],
 [8.0, 8.0, 4.0],
 [8.0, 4.0, 2.0],
 [4.0, 2.0, 2.0],
 [2.0, 2.0, 0.0],
 [2.0, 0.0, 0.0],
 [0.0, 0.0, 0.0],
 [0.0, 0.0, 2.0],
 [0.0, 2.0, 4.0],
 [2.0, 4.0, 2.0],
 [4.0, 2.0, 2.0],
 [2.0, 2.0, 2.0],
 [2.0, 2.0, 2.0],
 [2.0, 2.0, 2.0],
 [2.0, 2.0, 4.0],
 [2.0, 4.0, 4.0],
 [4.0, 4.0, 6.0]]

Y para y fue:
[8.0,
 8.0,
 8.0,
 8.0,
 8.0,
 8.0,
 4.0,
 2.0,
 2.0,
 0.0,
 0.0,
 0.0,
 2.0,
 4.0,
 2.0,
 2.0,
 2.0,
 2.0,
 2.0,
 4.0]
Ambas respuestas cumplen con las dimensiones solicitadas. Descarto un error como tal en los valores dentro de los arreglos teniendo en cuenta el mensaje que trae la nota en el que solo se menciona un error en las dimensiones. 
No se si hay otra forma en la que pueda mostrarles mejor el notebook y así puedan mirar mejor posibles errores.

Muchas gracias.

**Comment by Felipe-RA** [comment]
Saluidos davidlon

Recuerda que las dimensiones para la matriz X son `(len(ts)-n,3)` y para la matriz Y son `(len(ts)-n)`

Tienes un elemento menos del que deberías

PD: Recuerda hacer uso de las herramientas de visualización de código que abre el foro:

#145

**Reply by davidlon13** [reply]
Si ese era el error, muchas gracias.
